ui/webui/test/check-basic

```diff
@@ -61,7 +63,8 @@ class TestBasic(anacondalib.VirtInstallMachineCase):
         # Test that clicking on current step does not break navigation
         i.click_step_on_sidebar()
 
-        i.reach(i.steps.REVIEW)
+        # Create user on the first pass through users step
+        i.reach(i.steps.REVIEW, step_callbacks={i.steps.ACCOUNTS: lambda: create_user(self)})
```

Comment:
I think the callback setting should go to the installer as a default for ACCOUNTS step so that we can just call i.reach(i.steps.REVIEW) from the tests.

Reply:
Done in [webui: create required user when reaching a test step by default](https://github.com/rhinstaller/anaconda/pull/5280/commits/3bf083ad86de46a119a1cd9c7a221ca76127fff7)